docs/src/main/paradox/bootstrap/details.md

```diff
@@ -1,112 +1,40 @@
-# Bootstrap details
+# Initial bootstrap 
```

Comment:
Why change that to Initial? It describes the full process in more detail.

Reply:
Reverting